lua/nvim-tree/lib.lua

```diff
@@ -60,6 +60,12 @@ end
 
 local function get_node_at_line(line)
   local index = 2
+
+	local hide_root_folder = view.View.hide_root_folder
+	if hide_root_folder then
+		index = 1
+	end
```

Comment:
badly indented

Reply:
also for more concise reading:
`local index = view.View.hide_root_folder and 1 or 2`